### ISSUE TITLE
Unsandboxed Tor process "firefox.exe" cancelled by sandboxie

### ISSUE THREAD

**Issue by rugabunda** [sandboxie-plus/Sandboxie#1069]
**Describe the bug**
I sandbox firefox, and leave tor unsandboxed. When I load tor (which uses firefox.exe) launched outside of the sandbox sandboxie will cancel tors firefox.exe, rendering tor unusable, when you try open a new tab in Tor browser.

**To Reproduce**
1. Force sandboxing of standard firefox.exe
2. Click on 'disable forced programs'
3. Load the latest Tor browser
4. Open a New Tor Tab or, a few new tabs, and or try searching in a new tab
5. Sandboxie cancels firefox.exe, all new tabs will no longer function.

**Expected behavior**
Tor browser should just work

**Screenshots**

![SB](https://user-images.githubusercontent.com/38260267/127788872-d162c194-174f-40ac-8ecb-109b5576ba03.png)

**System details and installed software (please provide the following information):**
Windows 10 Enterprise 21H2).
sandboxie 0.9.1  

this bug began just today I believe, I believe in the previous version of sandboxie but many were released rapidly.

**Sandboxie configuration**

<details>
 
<summary>My Fireofx ini settings</summary>
 
```
firefox
Enabled=y
ConfigLevel=9
AutoRecover=y
BlockNetworkFiles=y
Template=FileCopy
Template=SkipHook
Template=Firefox_Profile_DirectAccess
Template=Firefox_Session_DirectAccess
Template=Firefox_Passwords_DirectAccess
Template=Firefox_Cookies_DirectAccess
Template=Firefox_Bookmarks_DirectAccess
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=Firefox_Phishing_DirectAccess
Template=AutoRecoverIgnore
RecoverFolder=%My Pictures%
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00ffff,off,6
BoxNameTitle=n
DropAdminRights=y
LeaderProcess=firefox.exe
NotifyStartRunAccessDenied=y
NeverDelete=n
NotifyInternetAccessDenied=y
ClosedFilePath=!<InternetAccess>,InternetAccessDevices
ClosedFilePath=F:\Tor Browser\
CopyLimitKb=81920
ForceProcess=firefox.exe
ProcessGroup=<StartRunAccess>,waterfox.exe,updater.exe,rundll32.exe,PotPlayerMini64.exe,plugin-container.exe,net.downloadhelper.coapp-win-64.exe,firefox.exe,ffprobe.exe,ffmpeg.exe,explorer.exe,DTDrop64.exe,dllhost.exe,crashreporter.exe,cmd.exe
ProcessGroup=<InternetAccess>,plugin-container.exe,net.downloadhelper.coapp-win-64.exe,firefox.exe,crashreporter.exe,crashr~1.exe
ProcessGroup=<ChromePrograms>,vivaldi.exe,opera.exe,neon.exe,maxthon.exe,iron.exe,dragon.exe
ProcessGroup=<FirefoxPrograms>,waterfox.exe,palemoon.exe
ClosedIpcPath=!<StartRunAccess>,*
ClosePrintSpooler=y
OpenSmartCard=n
BlockNetParam=y
OpenPrintSpooler=n
AllowSpoolerPrintToFile=n
CopyLimitSilent=y
AutoDelete=n
FakeAdminRights=n
AllowRawDiskRead=n
NotifyDirectDiskAccess=n

[GlobalSettings]

ClosePrintSpooler=y
OpenSmartCard=n
Template=RpcPortBindings
Template=7zipShellEx
Template=NOD32
Template=Microsoft_MSMQ
Template=OfficeLicensing
TemplateReject=ProxyCap
TemplateReject=WindowsLive
ForceDisableSeconds=10
FileRootPath=\??\%SystemDrive%\Sandbox\%USER%\%SANDBOX%
SeparateUserFolders=y
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
EditAdminOnly=n
ForceDisableAdminOnly=n
ForgetPassword=n

[UserSettings_02C600EE]

SbieCtrl_UserName=ty
SbieCtrl_NextUpdateCheck=-1
SbieCtrl_UpdateCheckNotify=y
SbieCtrl_ShowWelcome=n
SbieCtrl_WindowCoords=565,281,804,864
SbieCtrl_ActiveView=40021
SbieCtrl_ProcessViewColumnWidths=250,70,300
SbieCtrl_AutoApplySettings=y
SbieCtrl_SettingChangeNotify=n
SbieCtrl_ReloadConfNotify=n
SbieCtrl_HideWindowNotify=n
SbieCtrl_ExplorerWarn=n
SbieCtrl_TerminateWarn=n
SbieCtrl_ShouldDeleteNotify=n
SbieCtrl_EnableLogonStart=n
SbieCtrl_EnableAutoStart=n
SbieCtrl_AddDesktopIcon=n
SbieCtrl_AddQuickLaunchIcon=n
SbieCtrl_AddContextMenu=n
SbieCtrl_AddSendToMenu=n
SbieCtrl_EditConfNotify=n
SbieCtrl_HideMessage=2203,*GUIPROXY_00000001; MsgId: 18 - firefox.exe [00000102]
SbieCtrl_HideMessage=1308,RuntimeBroker.exe [Firefox]
SbieCtrl_HideMessage=1308,CompPkgSrv.exe [Firefox]
SbieCtrl_HideMessage=2101,AlpcConnectPort (C0000022) access=001F0001 initialized=1
SbieCtrl_HideMessage=1231,[C0000001 / 04]
SbieCtrl_HideMessage=2303,SetCurrentProcessExplicitAppUserModelID (33, 1655)
SbieCtrl_HideMessage=2318,cfgmgr32.dll
SbieCtrl_HideMessage=2303,CM_Add_Driver_PackageW (33, 1655)
SbieCtrl_HideMessage=2318,ole32.dll
SbieCtrl_HideMessage=2303,RegisterDragDrop (33, 1655)
SbieCtrl_HideMessage=2303,CoGetObject (33, 1655)
SbieCtrl_HideMessage=2101,ConnectPort (C0000022) access=001F0000 initialized=1
SbieCtrl_HideMessage=1301,firefox.exe
SbieCtrl_HideMessage=1301, vivaldi.exe
BoxDisplayOrder=Firefox,Vivaldi,VLC,Wireshark,Winamp,Binance,Chrome,F2,Flash,Malware_Box
SbieCtrl_BoxExpandedView=Chrome,F2,Firefox,Malware_Box,Vivaldi,VLC,Winamp,Wireshark
SbieCtrl_AutoStartAgent=SandMan.exe

[DefaultBox]

Enabled=y
ConfigLevel=9
AutoRecover=n
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00FFFF,ttl
Template=OpenBluetooth
Template=FileCopy
Template=SkipHook
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=Firefox_Phishing_DirectAccess
Template=AutoRecoverIgnore

[UserSettings]

SbieCtrl_AutoStartAgent=SandMan.exe
AutoRecover=n
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00FFFF,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=Firefox_Phishing_DirectAccess
Template=AutoRecoverIgnore
ConfigLevel=9

</details>

**Comment by Dyras** [comment]
I've had the same annoyance. My solution was to run Tor Browser in Whonix or Windows Sandbox, depending on which computer I'm at.

**Comment by DavidXanatos** [comment]
I think this is an issue of 0.9.1 it will be fixed in 0.9.2 and is iirc not present on 0.9.0
